helm/yatai-image-builder/templates/secret-yatai-common-env.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 kind: Secret
 metadata:
   name: {{ include "yatai-image-builder.yatai-common-envname" . }}
-  namespace: {{ .Values.yataiSystem.namespace }}
+  namespace: {{ .Release.Namespace }}
```

Comment:
Secret `yatai-common-env` needs to be under the namespace `yatai-system`, because `yatai-deployment/yatai-serverless` will also read this secret

Reply:
yatai-deployment & yatai-serverless is not installed in `yatai-system`, how can they read this secret?